resources/manifests/psp-privileged.yaml

```diff
@@ -63,3 +63,21 @@ subjects:
 - kind: Group
   name: system:serviceaccounts:kube-system
   apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: privileged-psp-kubelet
+  namespace: kube-system
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: privileged-psp
+subjects:
+- kind: Group
+  apiGroup: rbac.authorization.k8s.io
+  name: system:nodes
+- kind: User
+  apiGroup: rbac.authorization.k8s.io
+  # Legacy node ID
+  name: kubelet
```

Comment:
What is this for? Is it required? Do we know what it does? I think we should have a more detailed comment.

Reply:
I think we can get rid of this sub-section because in our case the nodes kubelet certs won't be ever use name `kubelet`, so removing it in subsequent PR.